src/thor/triplegbuilder_utils.h

```diff
@@ -172,7 +172,8 @@ struct MultimodalBuilder {
       TransitPlatformInfo* transit_platform_info = trip_node->mutable_transit_platform_info();
 
       // TODO: for now we will set to station for rail and stop for others
-      //       in future, we will set based on transitland value
+      //   not sure how to deal with this in the future: maybe assume it'll be
+      //   station for rail if there's a station and platform if not..
```

Comment:
here we're _always_ assuming it's a station when we travel by rail. however, we have the onestopid of the platform, not the station. that produces a mismatch between PBF & JSON response. old bug it seems.